plugins/node/opentelemetry-instrumentation-pg/src/semantic-conventions.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * These constants are considered experimental exports of `@opentelemetry/semantic-conventions`.
+ * They're being inlined until they're officially exported by `@opentelemetry/semantic-conventions`.
+ */
```

Comment:
This little comment will appear at the top of this file in other packages; let's workshop it if needed before it proliferates.